### ISSUE TITLE
Feature request. Disable double tapping stage/unstage

### ISSUE THREAD

**Issue by Batodalaev** [ForkIssues/TrackerWin#2062]
### Hell:
I accidentally double-click on a file and it moves between Staged and Unstaged, and back again.

When a lot of files have been changed (Staged and Unstaged both), it becomes difficult to check the changes before committing for the absence of unnecessary changes.

### Heaven:
I want stage/unstage files explicit only, by clicking a buttons Stage & Unstage (or context menu on right mouse click).

I think this behavior can be made optional and enable from settings.

**Comment by dracan** [comment]
Definitely a +1 for this! This drives me nuts! I'm always double-clicking the folder to expand it (as that's what normally happens in Windows with tree-views) - and it stages the entire folder! I think this is my biggest frustration with Fork - I have this issue all the time.

**Comment by The3dVehicleguy** [comment]
Agreed!!! I really would enjoy this becoming a feature seeing as sometimes my mouse double clicks (need to clean my mouse) or I have a lag spike due to an ongoing intensive project and it double clicks.

**Comment by ChristopherWalz** [comment]
I was about to buy a licence for fork, but without being able to change the behaviour for the double click, I will still have to wait.

**Comment by squall2022** [comment]
+1
